coderd/coderdtest/oidctest/idp.go

```diff
@@ -1009,6 +1050,21 @@ func (f *FakeIDP) httpHandler(t testing.TB) http.Handler {
 				http.Error(rw, "invalid code", http.StatusBadRequest)
 				return
 			}
+
+			if f.pkce {
+				challenge, ok := f.codeToChallengeMap.Load(code)
+				if !ok {
+					httpError(rw, http.StatusBadRequest, xerrors.New("code challenge not found for code"))
```

Comment:
```suggestion
					httpError(rw, http.StatusBadRequest, xerrors.New("challenge not found for code"))
```
This read a bit clearer to me, but if the original is more correct, keep it. Should we mention pkce here to give a hint `pkce: challenge ...`?